declaration/models.py

```diff
@@ -53,3 +54,16 @@ def __str__(self):
         return f'User: {self.user} - tracking code: {self.tracking_code}'
 
 
+class PackageStatusHistory(models.Model):
+    declaration = models.ForeignKey(PackageDeclaration, on_delete=models.CASCADE)
+    status = models.CharField(max_length=50)
```

Comment:
Wrong approach. You have model of statuses. We also need, an entry to be created automatically when package changes its status.